models/src_amazon_ads.yml

```diff
@@ -20,7 +20,7 @@ sources:
         identifier: "{{ var('amazon_ads_ad_group_history_identifier', 'ad_group_history') }}"
         columns:
           - name: _fivetran_synced
-            description: "{{ doc('_fivetran_synced') }}"\n
+            description: "{{ doc('_fivetran_synced') }}"\n 
```

Comment:
I noticed for all the `_fivetran_synced` fields there is a `\n` at the end. We should remove those.

Reply:
Whoops! done!